tests/fixtures_without_default/dag_factory.yml

```diff
@@ -0,0 +1,69 @@
+default:
+  concurrency: 1
+  dagrun_timeout_sec: 600
+  default_args:
+    end_date: 2018-03-05
+    owner: default_owner
+    retries: 1
+    retry_delay_sec: 300
+    start_date: 2018-03-01
+  default_view: tree
+  max_active_runs: 1
+  orientation: LR
+  schedule_interval: 0 1 * * *
+example_dag:
+  default_args:
+    owner: custom_owner
+    start_date: 2 days
+  description: this is an example dag
+  doc_md: '##here is a doc md string'
+  schedule_interval: 0 3 * * *
+  tasks:
+    task_1:
```

Comment:
Could we please represent the tasks as a list here since we now have PR #487 merged? There is a PR #492 attempting to convert all those tasks to a list in our example DAGs. So it would be nice if any new examples we add have them as lists.